requirements.txt

```diff
@@ -0,0 +1,33 @@
+annotated-types==0.6.0
+anyio==4.3.0
+certifi==2024.2.2
+click==8.1.7
+dnspython==2.6.1
+email_validator==2.1.1
+fastapi==0.110.2
+h11==0.14.0
+httpcore==1.0.5
+httptools==0.6.1
+httpx==0.27.0
+idna==3.7
+itsdangerous==2.2.0
+Jinja2==3.1.3
+MarkupSafe==2.1.5
+numpy==1.26.4
+opencv-python==4.9.0.80
+orjson==3.10.1
+pydantic==2.7.0
+pydantic-extra-types==2.6.0
+pydantic-settings==2.2.1
+pydantic_core==2.18.1
+python-dotenv==1.0.1
+python-multipart==0.0.9
+PyYAML==6.0.1
+sniffio==1.3.1
+starlette==0.37.2
+typing_extensions==4.11.0
+ujson==5.9.0
+uvicorn==0.29.0
+uvloop==0.19.0
+watchfiles==0.21.0
+websockets==12.0
```

Comment:
Consider using `rye` for easier setup of the environment. Happy to help if you need it! :)

Reply:
I love rye :100:

Reply:
💪